### DISCUSSION TITLE
v-model:selected="selected" error

### DISCUSSION THREAD

**Discussion by BalazsMakrai** [quasarframework/quasar#14876] [General - Components / Directives / etc] [answered]
I'm trying to create a table with multiple selections. I'm following the steps based on [this](https://quasar.dev/vue-components/table#selection). 
My code looks like this:
`<q-table :rows="rows" :columns="columns" row-key="name"  :selected-rows-label="getSelectedString" selection="multiple" v-model:selected="selected">...</q-table>`

And the setup function looks like this:
 setup() {
    const { t } = useI18n({}); 
    const selected = ref([]);
    };
  }

Somehow the ESLint throws an error: `'v-model' directives require no argument.`

How can I solve this?

**Comment by MilosPaunovic** [comment]
You could add another rule into your `eslintrc.js` file:
`"vue/no-v-model-argument": "off"`

**Reply by BalazsMakrai** [reply]
Thanks a lot!

**Comment by yusufkandemir** [comment, marked as answer]
Your ESLint setup is outdated, probably still for Vue 2. Create a new project using `yarn create quasar`/`npm init quasar`/etc. then mimic the setup by matching the ESLint-related package names, versions, etc., and the `.eslintrc.js` file.

If your project is a Vue 2 project(Quasar v1), then you should follow https://v1.quasar.dev instead. `v-model:selected` for Vue 3 is `:selected.sync` for Vue 2.
